practice-app/test/stock_test.js

```diff
@@ -0,0 +1,139 @@
+const chai = require('chai');
```

Comment:
`process.env.NODE_ENV = 'test';` line can be added to the top of the file so that when you run these tests from the ide, they again use the test database